docs/ref/django-admin.txt

```diff
@@ -979,6 +979,20 @@ Lets you pass a command as a string to execute it as Django, like so::
 
     django-admin shell --command="import django; print(django.__version__)"
 
+.. versionadded:: 1.11
+
+Lets you pass a script to execute via standard input on Linux and have it
+output to standard output, ie::
+
+    django-admin shell <<EOF
+    import django
+    print(django.__version__)
+    EOF
+
+Note that Windows select() implementation doesn't support file descriptors, so
+the output on Windows hosts will contain additional, interface specific
+outputs, making output parsing a lot harder.
```

Comment:
Maybe it should say something like

"On Windows the REPL is still outupt, due to limitations in its implementation of `select()`."

Output parsing is a second-order effect and probably doesn't need to be mentioned directly